gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/AuditInfoConverter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.remote.firestore.schema;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.android.gnd.model.AuditInfo;
+import com.google.android.gnd.model.Mutation;
+import com.google.android.gnd.model.User;
+import com.google.firebase.Timestamp;
+import java.util.Date;
+import java8.util.Optional;
+
+/** Converts between Firestore nested objects and {@link AuditInfo} instances. */
+class AuditInfoConverter {
+
+  /**
+   * Converts a POJO representing user and timestamp data in Firebase into an equivalent model
```

Comment:
Is it really a POJO? Doesn't it depend on Firebase?

Reply:
Good question... I *think* in this case they're still considered POJOs in the general sense in that they implement no special interfaces or external conventions; the annotations only provide hints for Firebase, but they're not required to use or create instances.

> The idea is that if the object (actually class) were a POJO before any annotations were added, and would return to POJO status if the annotations are removed then it can still be considered a POJO. Then the basic object remains a POJO in that it has no special characteristics (such as an implemented interface) that makes it a "Specialized Java Object" (SJO or (sic) SoJO).
https://en.wikipedia.org/wiki/Plain_old_Java_object#Definition

Reply:
Cool, sounds good! Thanks for clarifying. 

Might be best to just drop the term in this case? I don't think it adds much atm -- I think just calling it a converter to/from Firebase representation to the model is clear and effective.

Reply:
Done (9f16c32f4670c2d8313c87958891da0063c71c21)! I was going to update the comment, but ended up deleting instead, since other converter methods don't have method-level JavaDoc, and what they're doing is mostly clear from the class JavaDoc.